test/topology_custom/test_raft_recovery_basic.py

```diff
@@ -41,7 +41,7 @@ async def test_raft_recovery_basic(request, manager: ManagerClient):
     await asyncio.gather(*(delete_raft_data_and_upgrade_state(cql, h) for h in hosts))
 
     logging.info(f"Restarting {servers}")
-    await asyncio.gather(*(restart(manager, srv) for srv in servers))
+    await asyncio.gather(*(manager.server_restart(srv.server_id) for srv in servers))
```

Comment:
probably waiting logic doesn't work correctly when you fire it concurrently instead of rolling_restart?

`wait_others: int = 0`

Reply:
I don't see why not.